### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by joepetrowski**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+## PR Checklist
+
+- [ ] Are the audience and objective of the document clear? E.g. a document for developers that 
+  should teach them about transaction fees.
+- [ ] Is the writing:
+  - **Clear**: No jargon
+  - **Precise**: No ambiguous meanings
+  - **Concise**: Free of superfluous detail
+- [ ] Does it follow our [style guide](#documentation-style)?
+- [ ] If this is a new page, does the PR include the appropriate infrastructure, e.g. adding the
```
Comment (gnunicorn) [inline CONTRIBUTING.md:19]: this is a copy from a different repo, right? Because this doesn't have a sidebar, does it? ... Maybe talk about `the index` here?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+* [Documentation Standards](#documentation-standards)
+* [Documentation Style](#documentation-style)
+
+## PR Checklist
+
+- [ ] Are the audience and objective of the document clear? E.g. a document for developers that 
+  should teach them about transaction fees.
+- [ ] Is the writing:
+  - **Clear**: No jargon
+  - **Precise**: No ambiguous meanings
```
Comment (gnunicorn) [inline CONTRIBUTING.md]: As these articles go a lot into detail, I'd make these less restrictive. By all means "Digest vs. Inherents" is already full-on jargon just in the title. I'd rather make this:

```suggestion
  - **Clear**: All jargon defined, explained or linked to
  - **Precise**: No ambiguous meanings, rather specific links to definitions
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+  - **Concise**: Free of superfluous detail
+- [ ] Does it follow our [style guide](#documentation-style)?
+- [ ] If this is a new page, does the PR include the appropriate infrastructure, e.g. adding the
+  page to a sidebar?
+- [ ] Build the page. Does it render properly? E.g. no funny lists or formatting.
+- [ ] Do links go to rustdocs or devhub articles rather than code?
```
Comment (gnunicorn) [inline CONTRIBUTING.md]: is this good or bad if they do? The wording makes it unclear which is preferred...

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+
+If you are interested in contributing to Substrate Documentation, please follow these contribution 
+guidelines.
+
+* [PR Checklist](#pr-checklist)
+* [Documentation Standards](#documentation-standards)
+* [Documentation Style](#documentation-style)
+
+## PR Checklist
+
```
Comment (gnunicorn) [inline CONTRIBUTING.md:11]: If this is the PR checklist, shouldn't it be rather [be a PR template](https://help.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository)?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+developers who are working on Substrate. This directory includes documentation on how to get 
+started, build, run, and test Substrate. You should organize the content that you create by 
+specific activities, such as testing, getting started, or by workflow topic.
+
+### What should I document?
+
+Document protocols, introduce essential concepts, and explain how everything fits together.
+
+- Conventions: e.g. this document about documentation, code style
+- System Design: e.g. network stack, consensus, runtime, assumptions
```
Comment (gnunicorn) [inline CONTRIBUTING.md]:
```suggestion
- System Design: e.g. stack, subsystems, consensus, runtime, block format, assumptions
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+## Documentation Style
+
+It is important to create documentation that follows similar guidelines. This allows documentation
+to be clear and concise while allowing users to easily find necessary information.
+
+These are general style guidelines that can help create clearer documentation:
+
+- **Write in plain U.S. English.** You should write in plain U.S. English and try to avoid over-complicated words when you describe something.
+
+- **Avoid using pronouns such as "I" or "we".** These can be quite ambiguous when someone reads the
```
Comment (gnunicorn) [inline CONTRIBUTING.md:86]:
```suggestion
- **Use you rather than "I" or "we".** These can be quite ambiguous when someone reads the
```

The explanation says something else than the bold part...

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+clear explanation about how something works.**
+
+## Documentation Style
+
+It is important to create documentation that follows similar guidelines. This allows documentation
+to be clear and concise while allowing users to easily find necessary information.
+
+These are general style guidelines that can help create clearer documentation:
+
+- **Write in plain U.S. English.** You should write in plain U.S. English and try to avoid over-complicated words when you describe something.
```
Comment (gnunicorn) [inline CONTRIBUTING.md]: like the word over-complicated ;)

File: current/sidebar.json [hunk 8]
```diff
@@ -25,7 +25,10 @@
 		{
 			"title": "Integrate",
 			"slug": "integrate",
-			"menu_items": [ { "title": "Tools", "slug": "tools" }, { "title": "Front End", "slug": "front-end" } ]
```
Comment (gnunicorn) [inline current/sidebar.json:28]: a) this has a sidebar?
and b) this change seems unrelated

**Review by gnunicorn — APPROVED** [review]
minor nitpicks

File: CONTRIBUTING.md (hunk 1 above)
Comment (joepetrowski) [reply CONTRIBUTING.md:19]: > this is a copy from a different repo, right? 

A greatly reduced copy, but yes. Will leave the bullet point for now since forgetting this is a common mistake, but I'll come back to it whenever we know what the structure is like.

File: CONTRIBUTING.md (hunk 3 above)
Comment (joepetrowski) [reply CONTRIBUTING.md]: Breaking my own precision rules! Thanks.

File: CONTRIBUTING.md (hunk 4 above)
Comment (joepetrowski) [reply CONTRIBUTING.md:11]: Actually now it makes sense. We had this discussion when we first added this but decided against it since it doesn't apply to recipes/tutorials/etc. But now that it's just the KB, makes sense.

File: CONTRIBUTING.md (hunk 6 above)
Comment (joepetrowski) [reply CONTRIBUTING.md:86]: The explanation says "you" is acceptable. Personally, I still try to steer clear but sometimes it really is the best way. Changed it to be more clear.

File: current/sidebar.json (hunk 8 above)
Comment (joepetrowski) [reply current/sidebar.json:28]: Yes and yes, although I think (a) might be restructured depending on max's work.

Comment (joepetrowski) [reply current/sidebar.json:28]: I just noticed this while I was looking up (a) for myself and it bothered me :)
